src/backend/src/services/teams.services.ts

```diff
@@ -0,0 +1,25 @@
+import { Team } from 'shared';
+import { NotFoundException } from '../exceptions/NotFoundException';
+import prisma from '../prisma/prisma';
+import { teamTransformer } from '../transformers/teams.transformer';
+import { teamRelationArgs } from './prisma.relation.args/team.args';
+
+export class TeamsService {
```

Comment:
in this case I don't think the OOD aspect adds anything so I prefer functional

Reply:
We can't name the methods the same thing in the controller if we don't use the class. That's why I added it in the first place. What do you think the service methods should be named if not the same as their controller counterparts?

Reply:
in my pr i had like `editRiskController` in the controller file and `editRisk` in the services file

Reply:
I like this class way, because when it's called its super obvious that a service is being called (which imo makes this concept of separating the business logic out into a service easier for someone else to figure out), as opposed to the functional ones looking like any other function. I also like that we can name it the same thing because I think it makes it a little easier for someone new/ names like editRiskController are a bit confusing since they're not controllers, they're just methods in a controller

Reply:
@ShaanHossain @anthonybernardi

Reply:
I actually think of it the other way around where adding the boilerplate of classes doesn't add any extra functionality. I think that if we do it like this then the controllers should also be class based though